src/webview/token-cursor.ts

```diff
@@ -106,6 +107,26 @@ export class LispTokenCursor extends TokenCursor {
         return new LispTokenCursor(this.doc, this.line, this.token);
     }
 
+    /**
+     * Indicates if the current token is inside a string (e.g. a documentation string)
+     */
+    isInString() {
```

Comment:
I thought we already had a method for checking if we're inside a string? Or maybe that is only while parsing... But even if it is, maybe add that fact to the token?

Reply:
If so I didn't found it and the decision if in a string may depend on on the next and previous token so I think this method is correctly placed in the token cursor because only the cursor has the whole picture of the model.

Reply:
I was remembering correctly :smile:

https://github.com/BetterThanTomorrow/calva/blob/786d6dde336dfa53a91dcc1b6bc80eba99e71751/src/webview/token-cursor.ts#L414

Reply:
Hey cool! But I think the code is not sufficient. I will test it.

Reply:
What seems a bit odd to me is that double-clicking words work everywhere on in the REPL window, except the prompt. Is there a reason for that, you think? (I'm talking about the released Calva.)

Reply:
That's because outside the line model the default behavior of the double click is in place. We override the default for mouse events to set the blinking cursor on the right place. My new code takes care that this default behavior is preserved outside the line model.

Reply:
> My new code takes care that this default behavior is preserved outside the line model.

Yes. I saw that.